lib/pymedphys/_trf/decode/table.py

```diff
@@ -12,197 +12,350 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import List
+# from typing import List
 
 from pymedphys._imports import numpy as np
 from pymedphys._imports import pandas as pd
 
 from .constants import CONFIG
-from .header import determine_header_length
 
-GROUPING_OPTIONS = {
-    "integrity_v3": {"line_grouping": 700, "linac_state_codes_column": 2},
-    "integrity_v4": {"line_grouping": 708, "linac_state_codes_column": 6},
-    "unity_experimental": {"line_grouping": 700, "linac_state_codes_column": 6},
-}
+# from .header import determine_header_length
+
+# GROUPING_OPTIONS = {
+#     "integrity_v3": {"line_grouping": 700, "linac_state_codes_column": 2},
+#     "integrity_v4": {"line_grouping": 708, "linac_state_codes_column": 6},
+#     "unity_experimental": {"line_grouping": 700, "linac_state_codes_column": 6},
+# }
```

Comment:
No need to keep old code commented out. Really better to just delete it. It is still kept within the git history, so we can always find it if we ever need it.

Reply:
Once all the old code is deleted I'll be able to more easily do a diff based review of the changes. So I'll wait to review this file until after that change.

Reply:
I will delete the old code. and push again.